index.html

```diff
@@ -58,32 +50,29 @@
 
         const $ = {};
         window.$ = $;
+
+        // Create client configuration.
+        const configBuilder = Nimiq.Client.Configuration.builder();
         if (clientType === 'light') {
-            $.consensus = await Nimiq.Consensus.light();
-        } else if (clientType === 'nano') {
-            $.consensus = await Nimiq.Consensus.nano();
+            // Adding the Mempool feature triggers a light consensus.
+            configBuilder.feature(Nimiq.Client.Feature.MEMPOOL);
```

Comment:
Just a thought: this became a bit artificial now, in the UI the user can select light or nano, then the coder needs to know that asking for the mempool triggers a light client. How about we change the UI from "[x] pico [  ] light" to something like " [x] sync blockchain and maintain a local mempool - [  ] minimize data transfer, establish consensus quickly" (I'm not yet happy with the wording, but you get the idea - the question actually is, how much do we want to hide the client types?)

Reply:
I think it is fine the way it is, as this is also the way it was, and it should not be overcomplicated here.